bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -170,6 +170,12 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
 
   // Keys Management
   CONFIG_PARAM(keyExchangeOnStart, bool, false, "whether to perform initial key exchange");
+  CONFIG_PARAM(singleSignatureScheme,
+               bool,
+               true,
```

Comment:
default should be false until we decide to enable it

Reply:
doesn't matter, since the code which uses it is not included in this PR